cert_manager/__init__.py

```diff
@@ -0,0 +1,85 @@
+"""The main certificate manager class."""
+
+import logging
+
+from .utils import extract_x509_dns_names
```

Comment:
I prefer explicit relative imports over absolute imports, since they are less verbose and less repetitive, and allow moving things around within a package without adjusting all imports.  I'm not aware of any downsides of explicit relative imports.

Python 2 has _implicit_ relative imports (without the leading dot), which made it rather ambiguous how certain reports were resolved.  In Python 2, I recommend always using `from __future__ import absolute_import`, which will disable implicit relative imports.  Python 3 doesn't have them anymore anyway.

Reply:
Ah I remember reading about this, but that explains it a lot better. Sounds good then.